lib/infobox/link.js

```diff
@@ -21,6 +21,7 @@ define(['helper'], function (helper) {
 
     var arrow = document.createElement('span');
     arrow.classList.add('ion-arrow-right-c');
+    arrow.style.verticalAlign = 'middle';
```

Comment:
No style via CSS. I think the problem was different rendering on different platforms and with different fonts.

Reply:
👍 for @xf- comment